app/models/miq_policy_content.rb

```diff
@@ -3,6 +3,50 @@ class MiqPolicyContent < ApplicationRecord
   belongs_to :miq_event_definition
   belongs_to :miq_action
 
+  def self.seed
+    fixture_file = File.join(FIXTURE_DIR, "miq_policy_contents.yml")
+    policy_contents = File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
+
+    policy_contents.each do |content|
+      content.merge!(
+        :miq_policy           => MiqPolicy.find_by_name(content.delete(:miq_policy)),
+        :miq_event_definition => MiqEventDefinition.find_by_name(content[:miq_event_definition]),
+        :miq_action           => MiqAction.find_by_name(content[:miq_action])
+      )
+      policy_name = content[:miq_policy].try(:name)
+
+      rec = find_by_pkey(policy_name, content[:success_sequence], content[:failure_sequence])
+      if rec.nil?
+        create_seed_record(policy_name, content)
+      else
+        update_seed_record(rec, policy_name, content)
+      end
+    end
+  end
+
+  def self.create_seed_record(policy_name, content)
+    _log.info("Creating #{name}: [#{policy_name}, #{content[:success_sequence]} #{content[:success_sequence]}]")
+    MiqPolicyContent.create(content)
+  end
+
+  def self.update_seed_record(rec, policy_name, content)
+    rec.attributes = content
+    if rec.changed?
+      _log.info("Updating MiqPolicyContent: [#{policy_name}, #{content[:success_sequence]},"\
+                    " #{content[:failure_sequence]}]")
+      rec.save
+    end
+  end
+
+  def self.find_by_pkey(policy, success_sequence, failure_sequence)
+    relation = MiqPolicyContent.where(
+      :miq_policy       => policy,
+      :success_sequence => success_sequence,
+      :failure_sequence => failure_sequence
+    )
```

Comment:
@moolitayer 😕  I'm not familiar with that model.  It looks to be mostly unchanged since it was added, maybe @gtanzillo knows.